fenix/app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -327,6 +327,7 @@ class AwesomeBarView(
                     includeSponsoredSuggestions = state.showSponsoredSuggestions,
                     includeNonSponsoredSuggestions = state.showNonSponsoredSuggestions,
                     suggestionsHeader = activity.getString(R.string.firefox_suggest_header),
+                    contextId = activity.settings().contileContextId,
```

Comment:
Any chance that `contileContextId` is empty? Looks like it gets set over [here](https://github.com/mozilla-mobile/firefox-android/blob/644ec20c8f61a71a2686e109b4077fd2574e355a/fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt#L730-L734), which in turn gets called when Glean is initialized.

Just wanted to double check as I am not sure if we can assume that things we do here "happen after" the Glean initialization.

Reply:
Yep, Glean initializes right at startup; I think we can safely assume that Glean is initialized here, and `contileContextId` is non-empty, but let's record it in the ping, and double-check that it's set correctly.